tests/acceptance/01_vars/01_basic/os_name_human.cf

```diff
@@ -0,0 +1,64 @@
+body common control
+{
+  bundlesequence => { "test", "check" };
+}
+
+bundle agent test
+{
+  meta:
+      "description" -> { "CFE-3569" }
+        string => "Test for expected values of variable sys.os_name_human";
+
+  #
+  # Note:
+  #
+  # Order is important, the last variable assignment wins.
+  # E.g. prefer Ubuntu over Debian.
+  #
+  # This test should also succeed if none of the os-classes are defined.
+  # This should be covered in another test.
```

Comment:
@vpodzime @larsewi But it isn't covered in another test though, right? I would say that discovering suse is not defined on suse 15 was quite valuable, and we should cover this. Are you going to create a separate test for `sys.os_name_human != Unknown`? I think that's a valid test that we want to pass in all our tested platforms, always.

Reply:
I think it should be covered in a different test which I thought Lars was working on.

Reply:
@olehermanse @vpodzime Yes I'm planning doing that next. But I believe that this test belongs to [ENT-6873](https://tracker.mender.io/browse/ENT-6873) along with the fix.

Reply:
@vpodzime @olehermanse  I have instead written a test that checks wether or not `sys.os_name_human` resolves to `"Unknown"` here [PR 4554](https://github.com/cfengine/core/pull/4554). My comment above is not longer valid.